lib/get-selectors.js

```diff
@@ -8,6 +8,12 @@ function isNestedRule(node) {
   return /(?:at)?rule/.test(node.parent.type)
 }
 
+function isOnlyExtends(node) {
+  return node.nodes && node.nodes.length === 1 && node.every(function(child) {
+    return child.type === 'atrule' && child.name === 'extend'
+  });
+}
+
```

Comment:
On second thought I think your idea is better and the minimal overhead of two iterations allows the code to be clearer which I think is more important. Seeing as we now check the `node.nodes` in two places perhaps we could do:

```js
function hasChildNodes(node) {
  return node.nodes && node.nodes.length;
}

function hasNoDeclarations(node) {
  return hasChildNodes(node && node.every(function(child) {
    return child.type !== 'decl';
  });
}

function hasOnlyExtends(node) {
  return hasChildNodes(node) && node.every(function(child) { 
    return child.type === 'atrule' && child.name === 'extend';
  });
}
```

I was thinking that `has` instead of `is` reads a little better too?